gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/apim/core/shared_policy_group/model/SharedPolicyGroup.java

```diff
@@ -106,4 +106,24 @@ public static SharedPolicyGroup from(CreateSharedPolicyGroup createSharedPolicyG
             .steps(createSharedPolicyGroup.getSteps())
             .build();
     }
+
+    public static SharedPolicyGroup from(SharedPolicyGroup existingSharedPolicyGroup, UpdateSharedPolicyGroup sharedPolicyGroupToUpdate) {
```

Comment:
the naming is a bit confusing to me: the `ToUpdate` would be the entity to update, so the existing one.
what about `existingSharedPolicyGroup` and `updateSharedPolicyGroup` ?

Reply:
🚀